extension/src/wasm/bergamot-translator-worker.appendix.js

```diff
@@ -0,0 +1,203 @@
+/* global addOnPreMain, Module */
+
+addOnPreMain(function() {
+  let model;
+
+  const log = (message) => {
+    postMessage({
+      type: "log",
+      message,
+    })
+  }
+
+  const loadModel = () => {
+    log(`loadModel()`);
+    let start = Date.now();
+
+    // Delete previous instance if another model was loaded
+    if (model) {
+      model.delete();
+    }
+
+    // Set the Model Configuration as YAML formatted string.
+    // For available configuration options, please check: https://marian-nmt.github.io/docs/cmd/marian-decoder/
+    // This example captures the most relevant options: model file, vocabulary files and shortlist file
+    const modelConfig = `models:
+  - /model.esen.npz
+vocabs:
+  - /vocab.esen.spm
+  - /vocab.esen.spm
```

Comment:
Will the extension only support Es->En?

Reply:
No, you are right, I should bundle more models and make this dynamic.